CHANGELOG.md

```diff
@@ -7,8 +7,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 ### Added
 - Add in-memory docker image
 - Support of AWS S3 as blobstore
+- WebAdmin routes for CLI capabilities on mapping management (AddAddressMapping, AddRegexMapping, ListMappings)
```

Comment:
I don't understand this sentence. How does it relate to CLI?

Reply:
fixed